### ISSUE TITLE
Default config should use foreground color instead of white

### ISSUE THREAD

**Issue by jxerome** [dalance/procs#78]
Hello, 

I tried procs. 
But got a bad first time experience due to the default config that uses BrightWhite instead of Foreground color. 

Here is a capture:
<img width="1212" alt="Capture d’écran 2020-10-28 à 18 22 24" src="https://user-images.githubusercontent.com/1355634/97472834-8f74c180-194a-11eb-8f65-fe9e727f413f.png">

BrightWhite may be fancier for people with black background, but don't just work for others.

Thank you

**Comment by dalance** [comment]
Thank you for your request.
I think there are the following choices:

* change from BrightWhite to Foreground color by default
* set background color to black by default ( and add background color configuration )

I think the second choice is better because the same view can be provided to any background color.
What do you think?

**Comment by fwkz** [comment]
An ad-hoc workaround would be disabling colors using `procs -c disable`

**Comment by jxerome** [comment]
Well, I suppose using foreground would be the most user-friendly choice. Maybe using Bold color would keep BrightWhite for most people with dark themes.

Think about a tool that would change your background to white because that's the developer preference. I'm not sure you would like it. Especially setting a background different from terminal background is usually just ugly as it looks as if you highlighted the text.

The best scenario would be able to detect if the terminal uses a light or dark theme and automatically apply the correct color setting, but I don't know if it's only possible.

**Comment by dalance** [comment]
OK. I'll add auto detection of background color.
Some tools ( ex. vim/emacs ) detect the color on some terminals.

Now I'm working on https://github.com/dalance/termbg.

**Comment by jasikpark** [comment]
If there was at least a simple switch for light mode as well, that'd be swell.

Especially similar to `bat`:
```zsh
# .zshrc
alias bat="bat --theme=\$(defaults read -globalDomain AppleInterfaceSty  │ le &> /dev/null && echo default || echo GitHub)"
```

**Comment by Th3Whit3Wolf** [comment]
Somewhat off topic but being able to choose a config would be nice too. I use a daemon to change colorschemes at sunrise and sunset on linux. CLI programs that support this are great because I can place a script in `~/.local/bin/` with the same name of the file that chooses the conf based on time.

**Comment by dalance** [comment]
Sorry it took so long.
I added automatic theme detection, and released v0.11.0.

By default, dark/light theme is selected automatically.
If detection failed, you can specify theme by `--theme` option.

![image](https://user-images.githubusercontent.com/4331004/106115993-bedada80-6194-11eb-8713-8079e165bef4.png)
![image](https://user-images.githubusercontent.com/4331004/106116065-d4e89b00-6194-11eb-857d-ed13632b579b.png)

**Comment by dalance** [comment]
If you customized color by config file, some changes are required.
For example,

```diff
-style = "Black"
+style = "Black|White"
```

Please see for the detailed information:
https://github.com/dalance/procs#color-list

**Comment by ztlevi** [comment]
@dalance Hi, I tried the latest procs but the command column is still in grey color.

![image](https://user-images.githubusercontent.com/16655096/106344516-2c961c00-62e5-11eb-90d0-22d05d14bbc4.png)

![image](https://user-images.githubusercontent.com/16655096/106344495-10927a80-62e5-11eb-9b0c-27a340d92ef0.png)

Would you mind update that part? 

Thank you!

**Comment by dalance** [comment]
I forgot to apply light theme to default config.
I fixed it, and released v0.11.3.
